### ISSUE TITLE
loot overhaul 

### ISSUE THREAD

**Issue by SEBArma** [Sparker95/Vindicta#267]
remove all "recon" units from the loot script - that way, players will not be able to get better gear than the enemy as soon as they capture a single truck 

add grenades and rifle grenades to the loot tabes - we can take them form the units present, just like the other stuff 

add a slim chance of explosives to spawn in boxes 

reduce the overall amount of loot in trucks, not every truck should be completely filled with gear (especially not when acting as qrf/patrol

add more locations for boxes to spawn (if this can be done without automatically adding more boxes) to prevent boxes from spawning on roads

**Comment by Sparker95** [comment]
> add more locations for boxes to spawn (if this can be done without automatically adding more boxes) to prevent boxes from spawning on roads
Yes, just each outpost must currently have at least two such points. Need to ask @marvis999 to check Enoch map

**Comment by Sparker95** [comment]
Night Visions!! We need them for the night

**Comment by SEBArma** [comment]
for the vanilla factions, every soldier has NVGs 
in my factions only TL, SL and special forces have them - this is intentional - despite that we did not have trouble finding NVDs in our playtests

if we add grenades and rifle grenades as items that you can get from boxes, these can be used for fighting at night (flares, illum grenades)

**Comment by billw2012** [comment]
Yeah i like when NVG is scarce night combat with out them is pretty cool experience

**Comment by SEBArma** [comment]
- reduce the total amout of loot at an outpost by about 50% - right now it is way to easy to get an extreme amout of gear & equip hundreds of AIs 
- construction recources should not be reduced

**Comment by Sparker95** [comment]
I've added explosives, grenades and NVGs into cargo boxes.
What else do we need to change?

**Comment by billw2012** [comment]
Probably reduce loot over all, from what I gather there is enough currently so its never really a concern, but it should be a concern, and drive players to ambush the supply convoys.
